src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -54,44 +57,71 @@ limitations under the License.
     <define>OPERATIONAL_CREDENTIALS_CLUSTER</define>
     <description>This cluster is used to add or remove Operational Credentials on a Commissionee or Node, as well as manage the associated Fabrics.</description>
 
-    <attribute side="server" code="0x0001" define="FABRICS" type="ARRAY" entryType="FabricDescriptor" length="320" writable="false" optional="false">fabrics list</attribute>
-    <attribute side="server" code="0x0002" define="SUPPORTED_FABRICS" type="INT8U" writable="false" optional="false">SupportedFabrics</attribute>
-    <attribute side="server" code="0x0003" define="COMMISSIONED_FABRICS" type="INT8U" writable="false" optional="false">CommissionedFabrics</attribute>
+    <attribute side="server" code="0x0000" define="NOCs" type="ARRAY" entryType="NOCStruct" length="320" writable="false" optional="false">
+      <description>NOCs</description>
+      <access op="read" privilege="administer"/>
+      <access modifier="fabric-scoped"/>
+    </attribute>
+    <attribute side="server" code="0x0001" define="FABRICS" type="ARRAY" entryType="FabricDescriptor" length="320" writable="false" optional="false">
+      <description>Fabrics</description>
+      <access op="read" privilege="view"/>
+      <access modifier="fabric-scoped"/>
+    </attribute>
+    <attribute side="server" code="0x0002" define="SUPPORTED_FABRICS" type="INT8U" writable="false" optional="false">
+      <description>SupportedFabrics</description>
+      <access op="read" privilege="view"/>
+    </attribute>
+    <attribute side="server" code="0x0003" define="COMMISSIONED_FABRICS" type="INT8U" writable="false" optional="false">
+      <description>CommissionedFabrics</description>
+      <access op="read" privilege="view"/>
+    </attribute>
     <!-- 400 = 400 bytes for root cert -->
-    <attribute side="server" code="0x0004" define="TRUSTED_ROOTS" type="ARRAY" entryType="OCTET_STRING" length="400" writable="false" optional="false">TrustedRootCertificates</attribute>
-    <attribute side="server" code="0x0005" define="CURRENT_FABRIC_INDEX" type="fabric_idx" writable="false" optional="false">CurrentFabricIndex</attribute>
+    <attribute side="server" code="0x0004" define="TRUSTED_ROOTS" type="ARRAY" entryType="OCTET_STRING" length="400" writable="false" optional="false">
```

Comment:
The 400 length for an array here doesn't make sense

Reply:
This is what needs to be addressed as a group - pending on a group resolution :)
https://csamembers.slack.com/archives/G014G30SVV0/p1636739901137800?thread_ts=1636393190.014500&cid=G014G30SVV0

Reply:
This is the topic that needs group resolution :)